pkg/controller/node/nodecontroller.go

```diff
@@ -125,9 +127,10 @@ type NodeController struct {
 	// Lock to access evictor workers
 	evictorLock sync.Mutex
 	// workers that evicts pods from unresponsive nodes.
-	zonePodEvictor         map[string]*RateLimitedTimedQueue
-	zoneTerminationEvictor map[string]*RateLimitedTimedQueue
-	podEvictionTimeout     time.Duration
+	zoneIntolerablePodEvictor map[string]*RateLimitedTimedQueue
```

Comment:
I don't like the "intolerable" part of the name. IIUC it means that we can't stand the Pod, while in reality it's the Pod that can't stand taints. Maybe @davidopp will have a better suggestion.